contracts/governance/test/BaseTest.t.sol

```diff
@@ -86,31 +80,88 @@ contract BaseTest is Test {
         treasuryTimelock.grantRole(treasuryTimelock.CANCELLER_ROLE(), address(treasuryGovernor));
         vm.stopPrank();
 
-        return (
-            zrxToken,
-            token,
-            votes,
-            protocolTimelock,
-            treasuryTimelock,
-            address(protocolGovernor),
-            address(treasuryGovernor)
-        );
+        return (token, votes, protocolTimelock, treasuryTimelock, address(protocolGovernor), address(treasuryGovernor));
     }
 
-    function setupZRXWrappedToken() internal returns (IERC20, ZRXWrappedToken, ZeroExVotes) {
+    function setupZRXWrappedToken(IERC20 zrxToken) internal returns (ZRXWrappedToken, ZeroExVotes) {
         vm.startPrank(account1);
-        bytes memory _bytecode = vm.getCode("./ZRXToken.json");
-        IERC20 zrxToken;
-        assembly {
-            zrxToken := create(0, add(_bytecode, 0x20), mload(_bytecode))
-        }
-        address wTokenPrediction = predict(account1, vm.getNonce(account1) + 2);
+        address wTokenPrediction = predictAddress(account1, vm.getNonce(account1) + 2);
         ZeroExVotes votesImpl = new ZeroExVotes(wTokenPrediction, quadraticThreshold);
         ERC1967Proxy votesProxy = new ERC1967Proxy(address(votesImpl), abi.encodeCall(votesImpl.initialize, ()));
         ZRXWrappedToken wToken = new ZRXWrappedToken(zrxToken, ZeroExVotes(address(votesProxy)));
         vm.stopPrank();
 
         assert(address(wToken) == wTokenPrediction);
-        return (zrxToken, wToken, ZeroExVotes(address(votesProxy)));
+
+        return (wToken, ZeroExVotes(address(votesProxy)));
+    }
+
+    function mockZRXToken() internal returns (IERC20 zrxToken) {
+        vm.startPrank(account1);
+        bytes memory _bytecode = vm.getCode("./ZRXToken.json");
+        assembly {
+            zrxToken := create(0, add(_bytecode, 0x20), mload(_bytecode))
+        }
+        vm.stopPrank();
+    }
```

Comment:
Should we use `create` here or a combination of `vm.record`, `vm.etch`, and `vm.store` to get ZRX on its actual addres?

Reply:
This is only used in unit tests so I don't think we care the address it's deployed to as long as the bytecode's identical to production.